scripts/build-installer-win64.bat

```diff
@@ -2,19 +2,27 @@ rem DEPENDENCIES:
 rem   1. Node.js ('npm' binary in PATH)
 rem   2. 7zip    ('7z'  binary in PATH)
 rem
-rem   installer dev mode:  set SKIP_TO_FRONTEND/SKIP_TO_INSTALLER
+rem   installer dev mode:  set SKIP_TOOLS/SKIP_NODE/SKIP_CARDANO_FETCH/SKIP_STACK/SKIP_LIBS/SKIP_TO_FRONTEND/SKIP_TO_INSTALLER
+
+set /p CLUSTERS=<installer-clusters.cfg
+
+@echo ##############################################################################
+@echo ###
+@echo ### Will build clusters:  %CLUSTERS%
+@echo ###
+@echo ##############################################################################
 
 set MIN_CARDANO_BYTES=20000000
 set LIBRESSL_VERSION=2.5.3
 set CURL_VERSION=7.54.0
-set CARDANO_BRANCH_DEFAULT=cardano-sl-1.0
+set CARDANO_BRANCH_DEFAULT=release/1.1.0
```

Comment:
Note that this will conflict with https://github.com/input-output-hk/daedalus/pull/789/files so this change isn't needed.

Reply:
@domenkozar, that creates a kind of a problem -- you cannot really test the PR without that change -- and you cannot (auto-) merge with it included.  Heh : -)

Reply:
We could go with a manual conflict resolution merge commit pushed to this branch, by the way.  That'll probably close the PR automatically.

Reply:
You can always supply this as argument to the script, if I read it correctly?

Reply:
Sure, but punting the actual value to a different place won't make the conflict go away, which was our concern, FWIW.